docs/ROADMAP.md

```diff
@@ -1,44 +1,77 @@
 # kpt roadmap
 
-Last updated: May 6th, 2022
+Last updated: May 31st, 2022
 
-Please follow the [installation](https://kpt.dev/installation/) guide for installing the latest version of kpt.
+We added new components `package orchestrator`, `Config as Data UI`,
+`Config Sync` and `Go function SDK` to the `kpt` toolchain recently. We are
+looking for feedback for all the components that will help us prioritize work.
+If you have feedback about what you think we should be working on, we encourage
+you to get in touch (e.g. by filing an issue, or using the "thumbs-up" emoji
+reaction on an issue's first comment).
 
-We are planning to build a new roadmap pending user feedback on the new package orchestration capabilities.
+## 2022
 
-## Latest releases
+---
 
-Coming soon...
+## Areas of Focus
 
-## Detailed release notes
-Please refer to the [release notes] page for more information about the latest features.
-
-## Upcoming features
+---
 
 A few areas of work are ongoing. (This is not exhaustive.)
 
 ### Package orchestration
 
-See the [package orchestration roadmap](https://github.com/GoogleContainerTools/kpt/blob/main/porch/docs/porch-roadmap.md)
+This is an area where we want to support critical scenario associated with
+day 2 package management at scale, for example supporting bulk package update,
+restoring packages to previous versions, preview package changes, admission
+control at the repository level to enforce guardrails.
+
+See the [detailed package orchestration roadmap](https://github.com/GoogleContainerTools/kpt/blob/main/porch/docs/porch-roadmap.md)
+or see [the porch issues](https://github.com/GoogleContainerTools/kpt/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Fporch)
```

Comment:
Is there a separate porch github project?

Reply:
More generally, I'd expect this doc to link to projects in this and other kpt repos.